src/bentoml/bentos.py

```diff
@@ -419,3 +425,103 @@ def containerize(bento_tag: Tag | str, **kwargs: t.Any) -> bool:
     except Exception as e:  # pylint: disable=broad-except
         logger.error("Failed to containerize %s: %s", bento_tag, e)
         return False
+
+
+def serve(
+    bento: str,
+    production: bool = False,
+    port: int = BentoMLContainer.http.port.get(),
+    host: str = BentoMLContainer.http.host.get(),
+    type: str = "HTTP",
```

Comment:
avoid redefining reserved keywords. We should already have a lint rule for this.

```suggestion
    server_type: t.Literal["http", "grpc"] = "HTTP",
```

Reply:
Thanks for catching this.